DataProducts/DPP/Energy/Battery/LastChargeEntry_v0.1.json

```diff
@@ -0,0 +1,400 @@
+{
+  "openapi": "3.0.2",
+  "info": {
+    "title": "Last Charge Entry",
+    "description": "Information about latest charge cycle of a battery",
+    "version": "0.1.0"
+  },
+  "paths": {
+    "/DPP/Energy/Battery/LastChargeEntry_v0.1": {
+      "post": {
+        "summary": "Last Charge Entry",
+        "description": "Information about latest charge cycle of a battery",
```

Comment:
This is actually a bit wider than what the description has said earlier. I would widen this into "Information about the use of the battery" to describe the contents better